src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+interface Props {
+  todo: Todo;
+  onClose: () => void;
+  isLoading: boolean;
+}
+
+export const TodoModal: React.FC<Props> = ({ todo, onClose, isLoading }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    if (todo.userId) {
+      getUser(todo.userId).then(data => setUser(data));
```

Comment:
Consider adding error handling for the `getUser` API call. This will help manage potential errors gracefully and provide user feedback in the UI if the API request fails.